modules/post/multi/recon/local_exploit_suggester.rb

```diff
@@ -110,8 +110,9 @@ def valid_incompatibility_reasons(mod, verify_reasons)
   end
 
   def set_module_options(mod)
+    ignore_list = ['ACTION', 'TARGET'].freeze
```

Comment:
Context: The user setting `TARGET` to a value globally or locally in the datastore would unintentionally be assigned to the local exploit module that was being tested against